config/process_base.config

```
@@ -25,7 +25,7 @@ process {
     memory = {check_memory(48.GB  + 48.GB * task.attempt, params.max_memory)}
   }
   withLabel: mem_max {
-    memory = {task.attempt > 1 ? params.max_memory : check_memory(96.GB, params.max_memory)}
+    memory = {(task.attempt > 2  && task.exitStatus in 137..140) ? params.max_memory : check_memory(192.GB * task.attempt, params.max_memory)}
```

```suggestion
    memory = {(task.attempt > 1  && task.exitStatus in 137..140) ? params.max_memory : check_memory(192.GB * task.attempt, params.max_memory)}
```
I think if it's a memory failure on the first try then we want to increase the memory rather than run it again?

---

Ah okay, I didn't put together that by adding the `192.GB * task.attempt` you were still increasing the memory. This makes sense.